packages/fxa-settings/src/components/ConfirmWithLink/index.tsx

```diff
@@ -0,0 +1,117 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { ReactComponent as Mail } from './graphic_mail.svg';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+import { useFtlMsgResolver } from '../../models/hooks';
+
+export type ConfirmWithLinkProps = {
+  confirmWithLinkPageStrings: ConfirmWithLinkPageStrings;
+  email: string;
+  // TODO : update type definition once callback function is defined
+  goBackCallback?: () => void;
+  isOpenWebmailButtonVisible?: boolean; // TO-DO: Replace broker functionality which gives us this value (provider?)
```

Comment:
nit "TODO" (versus hyphenated flavor).
Least useful feedback ever, but helps w/ future grepping, plus we could potentially enable [`no-warning-comments`](https://eslint.org/docs/latest/rules/no-warning-comments) ESLint rule in the future that detect TODOs and FIXMEs [and possibly ensure that each TODO has a bug filed for it so it's tracked and gets to-done].

Probably not a huge deal, looks like we have about 16 instances of "TO-DO" throughout the code (in 9 files):
```sh
git grep -n "TO-DO" packages
git grep -l "TO-DO" packages | cat

packages/fxa-settings/src/components/ConfirmWithLink/index.test.tsx
packages/fxa-settings/src/components/ConfirmWithLink/index.tsx
packages/fxa-settings/src/pages/Signin/CompleteSignin/index.test.tsx
packages/fxa-settings/src/pages/Signin/CompleteSignin/index.tsx
packages/fxa-settings/src/pages/Signin/ConfirmSignin/index.test.tsx
packages/fxa-settings/src/pages/Signin/ConfirmSignin/index.tsx
packages/fxa-settings/src/pages/Signin/SigninBounced/index.tsx
packages/fxa-settings/src/pages/Signup/Confirm/index.test.tsx
packages/fxa-settings/src/pages/Signup/Confirm/index.tsx
```